.github/workflows/docker.yml

```
@@ -0,0 +1,23 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-docker:
```

Also, this makes the job name appear consistent with the other workflows. It would probably make sense to choose better names, but probably best to discuss that in another PR.

```suggestion
  build:
```